data/tribes/buildings/productionsites/barbarians/farm/init.lua

```diff
@@ -84,8 +84,8 @@ tribes:new_productionsite_type {
          descname = _"harvesting wheat",
          actions = {
             "callworker=harvest",
-            "animate=working 17000",
-            "sleep=3000"
+            "animate=working 3000",
+            "sleep=1000"
```

Comment:
I didn´t follow the discussion all that closely, but is it really such a huge difference needed?

Reply:
yes that was what had been consensus making them equal to empire Atlantean farms